modules/toybox/0.8.12.bcr.1/overlay/BUILD

```
@@ -0,0 +1,111 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
```

The native C++ rules have been removed in Bazel 9, please load them explicitly from `rules_cc`:

```suggestion
load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
```